addon/components/frost-button.js

```diff
@@ -1,14 +1,11 @@
+/**
+ * Component definition for the frost-button component
+ */
 import Ember from 'ember'
-const {
-  Component,
-  get,
-  isEmpty,
-  Logger,
-  typeOf,
-  ViewUtils
-} = Ember
+const {Component, Logger, ViewUtils, get, isEmpty, typeOf} = Ember
```

Comment:
I'm in agreement to use
```js
const {Component, computed, run} = Ember
const {readOnly} = computed
```
over
```
import {
  Component,
  computed: {
    readOnly
  },
  run
}  from 'ember'
```
as I find the latter much more eye bleeding. This is of course a personal preference.